src/mesh/generated/meshtastic/mesh.pb.h

```diff
@@ -187,6 +187,8 @@ typedef enum _meshtastic_HardwareModel {
     /* RadioMaster 900 Bandit, https://www.radiomasterrc.com/products/bandit-expresslrs-rf-module
  SSD1306 OLED and No GPS */
     meshtastic_HardwareModel_RADIOMASTER_900_BANDIT = 74,
+    /* AdaFruit Feather RP2040 RFM95 with optional SSD1306 display */
+    meshtastic_HardwareModel_RP2040_FEATHER_RFM95 = 75,
```

Comment:
These files are automatically generated. Changes would need to be added to the https://github.com/meshtastic/protobufs repository.

Reply:
Ah, I need to do Step 3 here:
https://meshtastic.org/docs/development/firmware/build/#distribute--publish-custom-builds

On the way...